script/jenkins/branches/run_unittest.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+export PATH=$PATH:$CONDAPATH
+
+source activate petals_env
+rm -rf tests_xml/
+rm -rf coverage/
+
+CORENV=~/jobs/petals_branches/core_env
+BRANCH=`git name-rev --name-only HEAD | cut -f 3- -d /`
+if [ -f $CORENV/$BRANCH ]; then
+    python -m venv --system-site-packages tvenv
+    source tvenv/bin/activate
+
+    pip install -e `cat $CORENV/$BRANCH`
+fi
+
+make unit_test PYTEST_CMD="python -m pytest"
```

Comment:
@emanuel-schmid This is how we can invoke a different command in the Makefile when executing it via Jenkins, see the changes in the Makefile as well

Reply:
nice!